bitbots_localization/include/bitbots_localization/ObservationModel.h

```diff
@@ -0,0 +1,117 @@
+//
+// Created by judith on 09.03.19.
+//
+
+#ifndef BITBOTS_LOCALIZATION_OBSERVATIONMODEL_H
+#define BITBOTS_LOCALIZATION_OBSERVATIONMODEL_H
+
+#include <particle_filter/ParticleFilter.h>
+#include <bitbots_localization/RobotState.h>
+#include <bitbots_localization/map.h>
+#include <bitbots_localization/tools.h>
+#include <humanoid_league_msgs/LineInformationRelative.h>
+#include <humanoid_league_msgs/GoalRelative.h>
+#include <humanoid_league_msgs/PixelsRelative.h>
+#include <humanoid_league_msgs/FieldBoundaryRelative.h>
+
+namespace hlm = humanoid_league_msgs;
+
+class RobotPoseObservationModel : public particle_filter::ObservationModel<RobotState> {
+
+public:
+
+    /**
+     * empty
+     */
+    RobotPoseObservationModel(std::shared_ptr<Map> map_lines, std::shared_ptr<Map> map_goals,
+                              std::shared_ptr<Map> map_field_boundary, std::shared_ptr<Map> map_corners,
+                              std::shared_ptr<Map> map_t_crossings, std::shared_ptr<Map> map_crosses);
+
+    /**
+     * empty
+     */
+    ~RobotPoseObservationModel();
```

Comment:
copy constructor, copy assignment operator, move constructor or move assignment operator. is not defined.